src/index.js

```diff
@@ -332,6 +333,8 @@ function addSlotToRoot (root, slot) {
   }
 
   rootToSlotMap.get(root)[slotName] = slot;
+  !slotToRootMap.has(slot) && slotToRootMap.set(slot, root);
```

Comment:
Because they are easily overread - they are definitely flagged in AUI. All good if they are not in named-slots